### ISSUE TITLE
A small mistake in file naming

### ISSUE THREAD

**Issue by QMike0** [Macavity/siyuan-database-properties-panel#33]
The Chinese version of the README is named as `README.zh_CN.md` incorrectly, which should be `README_zh_CN.md`

**Comment by Macavity** [comment]
Why is the filename relevant?

**Comment by QMike0** [comment]
Yeah, this filename is referenced in plugin.json as `"zh_CN": "README_zh_CN.md"`
![image](https://github.com/user-attachments/assets/2f312b7e-ae43-487e-868a-e0502298eb49)
So in Chinese version of SiYuan, If this file is not named in that way, there will be a mistake in plugin details interface:
![image](https://github.com/user-attachments/assets/00fa376c-e23e-4b29-9490-cda657b954c1)
and can not be found in `Downloaded Plugins` (Chinese version):
![image](https://github.com/user-attachments/assets/e77cdafe-eb78-4504-a1b6-9893b57e738f)

**Comment by Macavity** [comment]
Thank you for reporting this 👍 
I've fixed it in 1.0.3
